CONTRIBUTING.md

```diff
@@ -17,15 +17,18 @@ Adhere to the existing coding style and make sure to mimic best possible.
 
 To have a common look-and-feel [clang-format](https://clang.llvm.org/docs/ClangFormat.html)
 is used for code formatting. The formatting rules can be applied to the
-source code by running the following make target in your build directory:
+source code by running `make format` in your CMake build directory.
 
 ```sh
+$ mkdir -p build; cd build
+$ cmake ..
 $ make format
 ```
```

Comment:
As you may have guessed, I failed to realize to run 'make format' in the build directly. (First I tried 'make format' in the repo root directory and then .github/workflows/ci.yml which also doesn't explicitly write out the actual command to run. Then I tried clang-format but I missed to files in the tests directory.)

To root cause is of course that I haven't got used to the cmake indirection. In the Redis repo, make is used for everything directly in the repo root.

Reply:
ah, but this is a good change.
It seem that the text was added before the Makefile was re-added and updated, so at the time CMake was the only option.